app/controllers/application_controller.rb

```diff
@@ -1,20 +1,27 @@
-require_relative "../helpers/message_helper"
 class ApplicationController < ActionController::Base
   include MessageHelper
   protect_from_forgery with: :exception
 
-  helper_method :current_user, :logged_in
+  helper_method :current_user, :logged_in, :current_shop_owner
   before_action :set_categories_menu
 
   def set_categories_menu
     @menu_categories = Category.menu_categories
   end
 
   def current_user
-    @current_user ||= User.find_by_id(session[:user_id])
+    @current_user ||= RegularUser.find_by_id(session[:user_id])
```

Comment:
**Nitpick - Code Quality**
It is recommended that you make use of the `find_by` method in this case.

So you'd have something like this:

`RegularUser.find_by(id: session[:user_id)`

OR

`RegularUser.find(session[:user_id])`

depending on need.